src/commands/keys/get.ts

```diff
@@ -59,6 +60,12 @@ export default class GetEnvironmentKey extends Base {
         if (flags.env) {
             return flags.env
         }
+        const organization = this.repoConfig?.org || this.userConfig?.org
+        const project = this.repoConfig?.project || this.userConfig?.project
```

Comment:
Ideally we could display project/org property defined on the Base command. This command shouldn't need to know where the project/org came from (ie. repo vs user config), and it should always match what the base command is using, even if that logic changes in the future